R/sensitivity-analysis-run-options.R

```diff
@@ -13,17 +13,13 @@ SensitivityAnalysisRunOptions <- R6::R6Class(
     #' @param numberOfCores Number of cores to use for the simulation. Default value is \code{getOSPSuiteSetting("numberOfCores")}
     #' @param showProgress Should a progress information be displayed. Default value is \code{getOSPSuiteSetting("showProgress")}
     #' @return A new `SensitivityAnalysisRunOptions` object.
-    initialize = function(numberOfCores = ospsuiteEnv$numberOfCores,
-                              showProgress = ospsuiteEnv$showProgress) {
+    initialize = function(numberOfCores = NULL,

```

Comment:
This was missing here. Also I am differing initialization of number of cores  etc... to the function.
We had some issues in tlf or ospsuite::engine  and this also fixed some issues